### DISCUSSION TITLE
How to Quickstart Component Pack v7 with an existing (Win)WAS envoirnment ?

### DISCUSSION THREAD

**Discussion by MagnaFrisia90** [HCL-TECH-SOFTWARE/connections-automation#249] [Q&A] [answered]
Hello,
I have the following situation:
Connections v7 medium deployment on single Windows Server 2019 with DB2, DMGR & IHS.

I now want to use the ansible-playbooks to setup & integrate Component Pack into this envoirnment.
Can this be done with these playbooks ?

I have tried to run the example playbook "ansible-playbook -i environments/examples/cnx7/quick_start/inventory.ini playbooks/setup-component-pack-complete-harbor.yml" wich actually leads to the script trying to connect to my Windows-server DMGR via ssh.

![grafik](https://user-images.githubusercontent.com/115451939/217210877-8593b42b-de8c-454b-a8f8-f45b010b10d8.png)

**Comment by umeli** [comment]
Hi 
The harbor stuff is for CNX8. And from the readme:
Supported OSs:
CentOS 7+
RHEL 8+


with some tweaking it works with rocky 8

**Comment by stoeps13** [comment, marked as answer]
First use the playbook for component pack without Harbor, but I never tried to use this part on an already installed Connections WebSphere part. But I'm pretty sure, that this will not work because the playbook adds and overwrites settings and runs scripts on the DMGR and the scripts do not support Windows.

**Comment by MagnaFrisia90** [comment]
My expectation for this Github project was: HCL now offers the same Ansible as IBM did several years ago:  It will just install a Kubernetes envoirnment with Component Pack enabled which could be easily integrated into existing envoirnments.

As far as I understand your comments this is not the use case for this project. The whole project is only for new & linux envoirnments ? 
This is very disappointing :(

**Comment by stoeps13** [comment]
IBM never delivered Ansible roles for Connections, this started with HCL. All available roles and collection before were made by single people.

This is published as Open Source, so feel free to add this functionality. To be honest, I for example have no use case to add something to a manually installed environment. It needs more time to collect all paths and settings and add them to the environment file than a new installation on Linux needs.

There are still parts which need adjustments, so I wouldn't use this without tweaks in production. (Starting with running WebSphere as root...)

**Reply by MagnaFrisia90** [reply]
I am only talking about setting up  Kubernetes(IBM Cloud privat during these times). And I was hoping I could use these scripts to setup a single-node kubernetes with Component Pack as it worked with IBM Cloud private & the IBM offered helm charts during these days.
I will now just setup my k8s envoirnment manually

**Reply by stoeps13** [reply]
Cloud private setup is similar to create a cluster with kubeadm (Just one config file) , the helm charts are still available and heavily used in these roles.
